### ISSUE TITLE
Typescript support

### ISSUE THREAD

**Issue by spoon611** [zeroturnaround/sql-formatter#80]
Hi,

Can you publish the index.d.ts for this package so that its easily usable with typescript projects?

**Comment by kallaspriit** [comment]
Maybe this helps. Would be nice to either include it or make it available through DefinitelyTyped but have not gotten around to it.

```typescript
declare module "sql-formatter" {
  export interface FormatOptions {
    params?: unknown[];
    indent?: string;
    language?: "db2" | "n1ql" | "pl/sql" | "sql";
  }

  export function format(sql: string, options?: FormatOptions): string;
}
```

**Comment by toddpla** [comment]
Disclaimer: I have only recently started using typescript so this issues is probably something very elementary and most likely unrelated however when trying this inside a typescript file:
```ts
import sqlFormatter from 'sql-formatter';

sqlFormatter.format('...');
```
I get this error..
```sh
TypeError: Cannot read property 'format' of undefined
```
Any help would be appreciated

**Comment by toddpla** [comment]
> Disclaimer: I have only recently started using typescript so this issues is probably something very elementary and most likely unrelated however when trying this inside a typescript file:
> 
> ```ts
> import sqlFormatter from 'sql-formatter';
> 
> sqlFormatter.format('...');
> ```
> 
> I get this error..
> 
> ```shell
> TypeError: Cannot read property 'format' of undefined
> ```
> 
> Any help would be appreciated

It was an issue with my `tsconfig.json`  needed to include `"esModuleInterop": true`

**Comment by nsirbu** [comment]
https://www.npmjs.com/package/@types/sql-formatter

**Comment by nene** [comment]
Thanks.

**Comment by nene** [comment]
Type defs are now included in 4.0.1 release.
